documentation/man/features/report.rst

```diff
@@ -19,25 +19,46 @@ processing that will end up as a report for the user.
 
 OPTIONS
 =======
-\--day [<year>/<month>/<day>]:
+\--day[=<year>/<month>/<day>]:
   Display day summary. Users have the option to search a specific date by
   passing an argument that follows the *<year>/<month>/<day>* format. Not
   passing anything and gets info about today.
 
-\--week [<year>/<month>/<day>]:
+\--week[=<year>/<month>/<day>]:
   Shows the week summary. If a user does not pass any parameter kw will show
   the current week statistics. However, users can pass a random date
   *<year>/<month>/<day>* and let **kw** take care to provide a summary
   related to the week related to the target date.
 
-\--month [<year>/<month>]:
+\--month[=<year>/<month>]:
   This option shows a report regarding a specific month. Users can search for
   data related to a specific month by providing a parameter in the
   *<year>/<month>* format. If the user does not pass any parameter, **kw**
   displays the current month data.
 
-\--year [<year>]:
+\--year[=<year>]:
   Exhibits the current year summary if the user does not specify *<year>*.
 
 \--output <file-path>:
   Save the output of the report to *<file-path>*.
+
+EXAMPLES
+========
+For these examples, we suppose the fields in your **kworkflow.config** file are
+already configured.
```

Comment:
Maybe we can make this comment as a note and rewrite it as well. Maybe something like this:
> First of all, for these examples, we suppose that you have the `disable_statistics_data_track` option in your kworkflow.config enabled for a while.

Reply:
Also, keep in mind that this is only related to the `--statistics` option. In other words, this is not valid for the regular report; please, make this distinction explicit to the `--statistics` option.